### PR TITLE
Added steps to reset database after administrator added

### PR DESCRIPTION
**Pull request by tommycp3**

### CODE REVIEW

**Comment by zenweasel** [comment]
@tommycp3 Could you open this against `development` rather than `master`? Thanks.

File: developer/configuration.md [hunk 1]
```diff
@@ -25,6 +25,13 @@ For convenience, the initial Reaction administrator can be configured here.
 
 Creating a `settings.json` will prevent the default `dev.settings.json` from being loaded when you use the `reaction` command to start Reaction.
 
+Once you have edited the **_settings/settings.json_** you will need to run:
+```sh
+reaction reset && rc
```
Comment (zenweasel) [inline developer/configuration.md]: What is "rc" here?

Comment (tommycp3) [inline developer/configuration.md]: I'm not sure what rc means, it's what @aaronjudd  told me to run after modifying settings.json file, and I used it and it worked.

Comment (tommycp3) [inline developer/configuration.md]: I should have done this against development, my apologies.

Comment (aaronjudd) [inline developer/configuration.md]: my apologies to you both ;-) . `rc` is a bit of an undocumented shortcut for `reaction`.  They are interchangeable on the command line.   We _should_ mention that somewhere, and in the docs. however, in the example here we can just say `reaction`.  I guess the main thing was to mention the you need to reset, if you want to recreate the initial default user.

**Comment by aaronjudd** [comment]
@zenweasel, I updated the target PR branch, that's something we can do now, thank you github.

**Comment by zenweasel** [comment]
Closing and reopening with new branch because I can't seem to resolve conflict. There is nothing wrong with this PR

**Pull request closed without merging**